client/components/AddRatForm.js

```diff
@@ -6,12 +6,33 @@ import React from 'react'
 
 
 // Component imports
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { connect } from '../store'
+import { selectUser } from '../store/selectors'
 import Component from './Component'
+import classNames from '../helpers/classNames'
+import ValidatedFormInput from './ValidatedFormInput'
+import ValidatedFormSelect from './ValidatedFormSelect'
 
 
 
 
+// Component Constants
+const INVALID_NAME_MESSAGE = 'CMDR Name is Required'
+const INVALID_PLATFORM_MESSAGE = 'Platform is Required'
+const initialState = {
```

Comment:
name does not conform to constants naing scheme, shouldn't this be `INITIAL_STATE`?